enlighten/Controller.py

```diff
@@ -548,7 +555,8 @@ def connect_new(self, other_device=None):
         if isinstance(new_device_id.device_type, BLEDevice):
             pass
         elif str(new_device_id.address) != '111111' and str(new_device_id.bus) != '111111':
-            new_device_id.device_type = RealUSBDevice(new_device_id)
+            new_device_id.device_type = RealUSBDevice(new_device_id) # MZ: I don't like this
```

Comment:
Seeing the comments in Wasatch.PY, I will work on refactoring this so it is a proper implementation for distinguishing real vs sim devices